operator/src/main/java/oracle/kubernetes/operator/helpers/EventHelper.java

```diff
@@ -355,6 +360,11 @@ public String getReason() {
         return DOMAIN_COMPLETED_EVENT;
       }
 
+      @Override
```

Comment:
Wondering if it is important, certainly helpful, to make sure that all the pod cycling events that are associated with a roll are between the domain roll start event and domain roll complete event, say domain roll start, pod1 cycling, pod2 cycling, ..., domain roll complete.

Reply:
I saw that you only have an explicit ordering value for three event types. Not sure if you need it for roll start and pod cycling as well.

Reply:
Oh, I see. The events so ordered were generated simultaneously, and therefore we need to add them to Kubernetes in the appropriate order. Those other events are generated in separate steps and don't therefore need to be sorted.